<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplicationClass</class>
 <widget class="QMainWindow" name="QtWidgetsApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Addlayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="ResolveButton">
      <property name="text">
       <string>Resolve</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>listWidget</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>ResolveButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsApplicationClass</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ResolveButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsApplicationClass</receiver>
   <slot>resolve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsApplicationClass</receiver>
   <slot>remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>QtWidgetsApplicationClass</receiver>
   <slot>status()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>remove()</slot>
  <slot>resolve()</slot>
  <slot>slot1()</slot>
  <slot>status()</slot>
 </slots>
</ui>
